components-api/src/main/scala/pl/touk/nussknacker/engine/api/typed/TypeEncoders.scala

```diff
@@ -50,6 +50,10 @@ object TypeEncoders {
       val objTypeEncoded = encodeTypingResult(underlying)
       val tagEncoded = "tag" -> fromString(tag)
       objTypeEncoded.+:(tagEncoded)
+    case TypedObjectWithValue(underlying, data) =>
+      val objTypeEncoded = encodeTypingResult(underlying)
+      val dataEncoded = "data" -> SimpleObjectEncoder.encode(underlying, data).toOption.get
```

Comment:
Please don't use `Option.get`. It can cause scala's NPE (NoSuchElement Option.get). Better is be more explicit like `.getOrElse(illegal => throw new IllegalStateException(s"Not supported data type: $illegal")` or sth like this